src/components/Table/TableLoading.tsx

```diff
@@ -25,7 +25,7 @@ export const TableEmpty: FC<PropsWithChildren<TableEmptyProps>> = ({
   children,
 }) => {
   return (
-    <TableRow className="flex h-44 items-center rounded-t-none">
+    <TableRow className="flex h-48 items-center rounded-t-none">
```

Comment:
This perfectly matches the height the concentrator rewards panel will typically be